code/game/machinery/telecomms/ntsl2.dm

```diff
@@ -144,17 +173,54 @@ GLOBAL_DATUM_INIT(nttc_config, /datum/nttc_configuration, new())
 
 	// These two stack properly.
 	// Simple job indicator switch.
-	if(toggle_jobs)
-		var/new_name = signal.data["name"] + " ([signal.data["job"]]) "
-		signal.data["name"] = new_name
-		signal.data["realname"] = new_name // this is required because the broadcaster uses this directly if the speaker doesn't have a voice changer on
+	if(job_indicator_type)
+		var/new_name = signal.data["name"]
+		var/job = signal.data["job"]
+		var/job_color = "#000000"
+		if(job in jobs_ai)
+			job_color = "#FF00FF"
+		if(job in jobs_command)
+			job_color = "#204090"
+		if(job in jobs_engineering)
```

Comment:
Change those checks to else if and order them by the order you'd like them to override eachother - Telecomms is a performance heavy thing and avoiding unnecessary if check is good practice.

Reply:
This I will do